src/zmq/zmqutil.h

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2014-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_ZMQ_ZMQUTIL_H
+#define BITCOIN_ZMQ_ZMQUTIL_H
+
+void zmqError(const char *str);
```

Comment:
49e2cb78563df27ff0a5d2ad985e074ee98afcce
nanonit:
```suggestion
void zmqError(const char* str);
```